src/main/java/org/opentripplanner/graph_builder/module/DirectTransferGenerator.java

```diff
@@ -82,23 +81,22 @@ public void buildGraph(
 
             LOG.debug("Linking stop '{}' {}", ts0.getStop(), ts0);
 
```

Comment:
Not really related to this pull request, as the functionality was the same before. I'm just wondering about the rationale behind this logic. A stop that already has at least one outgoing simpleTransfer (created by pathways) should not have a simpleTransfer generated?

Reply:
Looks like this comes from a time prior to the refactoring to use RATOR, as only the SimpleTransfers are used for transferring. The NearbyStopFinder will traverse pathways when looking for potential transfers, so they should be ok. Removed the check.